packages/apidom-ls/src/services/validation/linter-functions.ts

```diff
@@ -1068,4 +1069,41 @@ export const standardLinterfunctions: FunctionItem[] = [
       return true;
     },
   },
+  {
+    functionName: 'apilintOpenAPIParameterFieldIsDefinedWithinPathTemplate',
+    function: (element: Element) => {
+      if (element.element === 'parameter') {
+        const parameterLocation = toValue((element as ObjectElement).get('in'));
+        const isInPathItemElement =
+          isArrayElement(element.parent) &&
+          includesClasses(['path-item-parameters'], element.parent);
+
+        if (!isInPathItemElement || parameterLocation !== 'path') {
```

Comment:
Just an optimization tip (which is up to you to utilize):

if you bail out directly after:

```js
const parameterLocation = toValue((element as ObjectElement).get('in'));

if (parameterLocation !== 'path') {
  return true;
}
```

You'll avoid `isInPathItemElement` assertions which does perform some ApiDOM conversions under the hood which might slightly hit performance.